app/models/challenge.rb

```diff
@@ -5,4 +5,13 @@ class Challenge < ApplicationRecord
   has_many :challenge_assignments, dependent: :destroy
   has_many :batchs, through: :challenge_assignments
 
+  validates :score, presence: true, 
+                    numericality: { 
+                      only_integer: true, 
+                      greater_than_or_equal_to: 1, 
+                      less_than_or_equal_to: 10 
+                    }
+  validates :title, presence: true
+	validates :description, presence: true
```

Comment:
Nice job on the score validation here! Line 15's indentation seems a little off, though 😅